sympy/simplify/simplify.py

```diff
@@ -74,6 +74,9 @@ def separatevars(expr, symbols=[], dict=False, force=False):
     Examples
     ========
 
+    >>> from sympy import simplify, sin, rad
+    >>> abs((simplify(30.8**2 - 82.5**2 * sin(rad(11.6))**2)).evalf())
+    >>> assert _-673.447451402970 < 1e-12
```

Comment:
The actual value is 673.295830164072 so it looks like simplify is using the `to/from_sympy` which is odd.